README.md

```diff
@@ -68,39 +69,34 @@ drush rsync @production:%files @self:%files
 
 ### Working on the theme
 
-The hatter designs are in a Butler styleguide at styleguide/. To use it, first you'll need to install. You can run the styleguide on either your host or your vm, but only the environment you install on. It's probably easiest on your host.
+As of 2019, component markup, styles and interactivity all live in the [Hatter Styleguide](https://github.com/MidCamp/hatter-v2)
+The majority of theming work will occur in that repository and instructions are included in the [readme](https://github.com/MidCamp/hatter-v2/blob/master/README.md). 
 
-```
-cd styleguide
-npm install
-```
-
-## What does this Butler do?
-
-Butler is tool to automate front-end development tasks and streamline prototyping. Run npm commands from the `styleguide/` directory:
-
-* `npm run butler`
-
-  This is the default task. This will watch your sass/spress files for changes and compile/build accordingly. It will also flag any sass linting errors before compiling. It will output CSS that has been been minified and optimized. While butler is running, the styleguide is available at http://midcamp.local:4000.
+The compiled assets are packaged as an npm dependency. To install the latest, run `npm install` either in the root of this
+repository, or the theme directory. The initial setup process, along with the Circle CI build process will install these
```

Comment:
Circle CI is builds and tests the application, but it's not actually sending artifacts anywhere to be used.  We'll need to install these in Lagoon too.